### PR TITLE
parse root auth attribute from collection

### PR DESCRIPTION
**Pull request by gustavocortarelli**
get the auth attribute from JSON collection and parse it to the auth root collection attribute

### CODE REVIEW

File: tests/test_collection_auth.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                "name": "200 ok",
+                "request": "http://echo.getpostman.com/status/200",
+            }
+        ],
+    }
+    collection = Collection()
+    collection.parse(coll)
+    assert collection.auth is not None
+    assert collection.auth.auth_type == AuthType.APIKEY.value
+    for auth_type in AuthType:
```
Comment (kirankumbhar) [inline tests/test_collection_auth.py:73]: You don't need `for` loop to again assert `collection.auth.auth_type`. Which you have already checked in the previous line.

Remove this for loop. Instead, check for the `len(collection.auth.apikey)` is 2 and both `AuthAttribute` objects has required `key` attribute according to postman schema.

**Review by kirankumbhar — CHANGES_REQUESTED** [review]

**Review by kirankumbhar — APPROVED** [review]

**Comment by kirankumbhar** [comment]
Thank you @gustavocortarelli for fixing this!

**Comment by subho007** [comment]
@gustavocortarelli thank you for the contribution. Released - https://github.com/appknox/postmanparser/releases/tag/v0.1.5 also in `PyPi`

**Comment by gustavocortarelli** [comment]
thanks to you all for this helpful project =]

**Pull request merged**